docs/2023/copyrights/updates/2023-05-31.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Week 1
+author: Abdelrahman Jamal
+---
+<!--
+SPDX-License-Identifier: CC-BY-SA-4.0
+
+SPDX-FileCopyrightText: 2023 Abdelrahman Jamal <abdelrahmanjamal5565@gmail.com>
+-->
+
+*(May,31,2023)*
+
+## Attendees:
+
+* [Abdelrahman](https://github.com/Hero2323)
+* [Anupam](https://github.com/ag4ums)
+* [Ayush](https://github.com/hastagAB)
+* [Gaurav](https://github.com/GMishx)
+* [Kaushlendra](https://github.com/Kaushl2208)
+
+## Updates:
+- We had some minor discussions about the content of the proposal.
+- We discussed the feature timeline to be followed on the project and as I had finals starting on the 28th of May and ending on the 13th of June, everyone agreed that for weeks 1 & 2 of the coding period, only minor fixes to the previous project as implemented by Kaushlendra will be made.
+- we had a discussion about the creation of the new dataset to be used and that it will created for at least one large open source project [Fossology](https://github.com/fossology/fossology) and possibly other large open source projects as well.
```

Comment:
```suggestion
- We had a discussion about the creation of the new dataset to be used and that it will created for at least one large open source project [Fossology](https://github.com/fossology/fossology) and possibly other large open source projects as well.
```